qlib/Swagger.qm

```diff
@@ -1154,12 +1157,15 @@ public class SwaggerSchema inherits ObjectBase, AbstractRestSchemaValidator {
             }
             else {
                 *code ds = MimeDataTypes.(headers."content-type").deserialize;
-                if (!ds)
-                    throw "DESERIALIZATION-ERROR", sprintf("%s %s: cannot deserialize message body with Content-Type: %y; supported MIME types: %y; available deserialization modules: %y", method.upr(), path, headers."content-type", MimeContentTypes, SerializationModules);
-
-                auto any_body = ds(http_body);
-                if (exists any_body)
-                    body = any_body;
+                if (!ds) {
+                    #throw "DESERIALIZATION-ERROR", sprintf("%s %s: cannot deserialize message body with Content-Type: %y; supported MIME types: %y; available deserialization modules: %y", method.upr(), path, headers."content-type", MimeContentTypes, SerializationModules);
```

Comment:
what should we do with this line?   I assume we should remove it, however somewhere we need to check that the content-type for the message received is acceptable by the method according to the schema - I think this is missing now with this change, correct?

Reply:
I would remove it. Or probably it could be changed to account for the case when content-type header is missing, then it could throw something. But otherwise, if the content-type does not match one of the default types, then I would just pass the raw binary to the service code to handle it by itself.